nevergrad/optimization/optimizerlib.py

```diff
@@ -2539,6 +2539,18 @@ def _num_objectives_set_callback(self) -> None:
                 # override at runtime
                 self._optim = DE(self.parametrization, self.budget, self.num_workers)
 
+    def recommend(self) -> p.Parameter:
```

Comment:
please don't copy paste code, otherwise we'll forget to update in both locations if we have to update this function.
here you can use `Optimizer.recommend(self)`
Also, don't you want to integrate this in `NGOptBase` instead? That way all variants can benefit it.
Finally, remove the docstring, the one that matters is the one in the base class

Reply:
I do another PR with this change alone first.

Reply:
Moved to another PR.